### DISCUSSION TITLE
Aerial

### DISCUSSION THREAD

**Discussion by martinpallmann** [nvim-telekasten/telekasten.nvim#314] [Q&A] [answered]
Hi, is anyone using an outliner like [aerial.nvim](https://github.com/stevearc/aerial.nvim/tree/master) successfully with telekasten? And if so, how?

**Comment by petrisch** [comment]
What would you expect?
I use Aerial and Telekasten side by side, but Aerial is just acting on the Markdown files titles.
I use it rarely because I try to keep the files short.

**Reply by martinpallmann** [reply]
I would expect it to show my outline. But it doesn't, since the filetype of Telekasten is 'telekasten' and not 'markdown', so either I'd need to make Aerial understand the telekasten filetype or change the filetypes of my telekasten files to markdown. I'm not sure how either of these things work and what the implications are.

**Reply by lambtho12** [reply, marked as answer]
If you put `auto_set_filetype = false` in telekasten, it should not set the filetype as telekasten. 
This telekasten filetype has very limited use in general. It is mostly used to save automatically telekasten files before renaming a note and updating links or allow specific snippets for telekasten and not regular markdown files for instance. This should be about it.
